tests/e2e/config/wordpress-debug-log/index.js

```diff
@@ -140,7 +140,7 @@ async function assertEmptyDebugLog() {
 	// Wait 1 second for any log data to finish propagating.
 	// Without this, node can disconnect from the log stream
 	// before the entries are recorded and result in a false success.
-	await page.waitForTimeout( 1000 );
+	await new Promise( ( resolve ) => setTimeout( resolve, 1000 ) );
```

Comment:
I think these changes to the E2E tests can be reverted, using `page.waitForTimeout()` is working fine for me locally.

Reply:
Yup, this might not be needed; I thought I removed it but I guess I missed these. I'll go back and revert if it works 👍🏻